src/app/features/task-view-customizer/task-view-customizer.service.ts

```diff
@@ -0,0 +1,255 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, combineLatest } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { TaskWithSubTasks } from '../tasks/task.model';
+import { selectAllProjects } from '../project/store/project.selectors';
+import { selectAllTags } from './../tag/store/tag.reducer';
+import { Store } from '@ngrx/store';
+import { Project } from '../project/project.model';
+import { Tag } from '../tag/tag.model';
+
+@Injectable({ providedIn: 'root' })
+export class TaskViewCustomizerService {
+  public selectedSort$ = new BehaviorSubject<string>('default');
+  public selectedGroup$ = new BehaviorSubject<string>('default');
+  public selectedFilter$ = new BehaviorSubject<string>('default');
+  public filterInputValue$ = new BehaviorSubject<string>('');
```

Comment:
Using signals for this would allow us to not use the getter functions in the view, which possibly is more performant.